### ISSUE TITLE
Doesn't work for "noname" buffers

### ISSUE THREAD

**Issue by habamax** [RRethy/vim-hexokinase#4]
If there is no buffer name (for example when you have turned off :intro in your vimrc):

    set shortmess+=I

And you input a color to that buffer:

    #ffff00

Then run `:HexokinaseToggle` to get error:

![image](https://user-images.githubusercontent.com/234774/54978421-62fc0500-4fb1-11e9-96cb-b5a6c2de66ad.png)

If you run it second time you will get:

![image](https://user-images.githubusercontent.com/234774/54978546-ba9a7080-4fb1-11e9-9b2c-680b835febd6.png)

**Comment by RRethy** [comment]
Nice catch!

**Comment by habamax** [comment]
Hi, still have an error:

![image](https://user-images.githubusercontent.com/234774/55055949-b0da4100-5075-11e9-97cb-46feda86b8e6.png)

1. Open vim without intro
2. add `hello #ff00ff world #ff0000` line
3. `:HexokinaseToggle` -- OK
4. change line to `hello #ff00ff world #0000ff`
4. `:HexokinaseRefresh` -- Error

**Comment by RRethy** [comment]
Sorry just saw that you had posted a reply, I didn't get notified in my email for some reason. In future reopening the issue will make it more likely the author see the reply. I just tried recreating the behaviour but can't, do it still persist?


Also, the main message I was posting in my issues lol:
I just released v2 of vim-hexokinase. It's scrapes asynchronously and has 6 different methods for displaying the colours in a file!

To upgrade, run `git pull && make hexokinase`. For more information on migrating to v2, check out `:h hexokinase-v1-migration`. There are a few fully deprecated options (`:h hexokinase-v1-deprecation`) but almost everything has a v2 alternative. v1 will still work and can be forced by using `:let g:Hexokinase_v2 = 0`.
